src/components/NostoSkuOptions.ts

```diff
@@ -122,6 +126,10 @@ function registerClickEvents(optionId: string, { selectSkuOption }: Store, optio
       option.toggleAttribute("selected", true)
       optionElements.filter(o => o !== option).forEach(o => o.removeAttribute("selected"))
       selectSkuOption(optionId, skuIds)
+
+      const img = option.getAttribute("ns-img")
+      const altImg = option.getAttribute("ns-alt-img")
+      setSkuImages?.(img || undefined, altImg || undefined)
```

Comment:
we don't want to override anything if img and altImg are empty

so maybe wrap this with
if (img) { ...